src/Behat/MinkExtension/Context/RawMinkContext.php

```diff
@@ -158,7 +158,7 @@ public function saveScreenshot($filename = null, $filepath = null)
     {
         // Under Cygwin, uniqid with more_entropy must be set to true.
         // No effect in other environments.
-        $filename = $filename ?: sprintf('%s_%s_%s.%s', $this->getMinkParameter('browser_name'), date('c'), uniqid('', true), 'png');
+        $filename = $filename ?: sprintf('%s_%s_%s.%s', $this->getMinkParameter('browser_name'), date('Y-m-d_H.i.s'), uniqid('', true), 'png');
```

Comment:
@aik099 I'm not sure why that would be better. It would increase the amount of processing needed when we have full control over the filename anyway.